installer/scripts/openshift/setupKeptn.sh

```diff
@@ -130,3 +130,8 @@ verify_kubectl $? "Deploying keptn api virtualservice failed."
 
 helm init
 oc adm policy  add-cluster-role-to-user cluster-admin system:serviceaccount:kube-system:default
+oc adm policy add-scc-to-group privileged system:serviceaccounts -n keptn
+oc adm policy add-scc-to-group anyuid system:serviceaccounts -n keptn
+
+# Inject istio into jmeter-service
+kubectl get deployment -n keptn jmeter-service -oyaml | istioctl kube-inject -f - | kubectl apply -f -
```

Comment:
As as with the installer above, if we ever use another service, e.g., "neoload", we would have to rewrite this.

Reply:
istio injection is not needed in the jmeter service anymore since we use the *-canary service URL in the jmeter service.